lighthouse-cli/test/cli/cli-flags-test.js

```diff
@@ -6,13 +6,18 @@
 'use strict';
 
 /* eslint-env jest */
-const assert = require('assert').strict;
-const getFlags = require('../../cli-flags.js').getFlags;
+
+import {strict as assert} from 'assert';
+import fs from 'fs';
```

Comment:
Do we need to be consistent with `import fs from 'fs'` or `import * as fs from 'fs`?

Reply:
`fs` is commonjs and so has no default export, so the entire module (aka `* as fs`) is treated as the default.

I have no idea which to go with, my first thought is leaning towards using the default import style.

Reply:
`import fs from 'fs'` is my preferred version by far